spec/e2e/scope_spec.rb

```diff
@@ -26,6 +29,17 @@
     delete_db
   end
 
+  describe 'Inherited scope' do
+    before { stub_named_class('Mutant', Person) }
+
+    let(:alive) { Mutant.create name: 'aa' }
+    let(:dead)  { Mutant.create name: 'bb', date_of_death: 'yesterday' }
+
+    it 'has the scope of the parent class' do
+      expect(Mutant.scope?(:only_living)).to be true
+      expect(Mutant.as(:m).only_living(:m).to_a).to eq([alive])
```

Comment:
There's no need for the node variable, I think.  You could just do `Mutant.all.only_living.to_a`.  You might even be able to drop the `to_a` because I think when it compares `eq` it will enumerate it.